pallets/anchors/src/lib.rs

```diff
@@ -292,10 +293,10 @@ pub mod pallet {
 			}
 
 			// pay the state rent
-			let today_in_days_from_epoch =
-				TryInto::<u64>::try_into(<pallet_timestamp::Pallet<T>>::get())
-					.map(common::get_days_since_epoch)
-					.or(Err(Error::<T>::FailedToConvertEpochToDays))?;
+			let now_u64 = TryInto::<u64>::try_into(<pallet_timestamp::Pallet<T>>::get())
+				.or(Err(Error::<T>::FailedToConvertEpochToDays))?;
```

Comment:
FailedToConvertEpochToDays and EvictionDateTooBig errors could mean the same?

Reply:
Yeah, I will reuse ArithmeticError::Overflow